compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -184,11 +184,13 @@ void addGPUMatmulTensorCorePassPipeline(OpPassManager &pm,
 }
 
 void addGPUWarpReductionPassPipeline(OpPassManager &pm) {
-  tileAndBufferize(pm);
-
-  // Don't tile to warp or thread and use vector distribution instead.
-
+  pm.addPass(createTileAndDistributeToWorkgroupsPass());
   auto &nestedModulePM = pm.nest<ModuleOp>();
+  nestedModulePM.addNestedPass<func::FuncOp>(
+      createConvertToDestinationPassingStylePass());
+  nestedModulePM.addPass(createCanonicalizerPass());
```

Comment:
We have back-to-back (canonicalizer + CSE)s. Is it intended?

Reply:
ah no, not sure how that happened